rules/CodeQuality/NodeAnalyser/AssertMethodAnalyzer.php

```diff
@@ -34,28 +35,50 @@ public function detectTestCaseCall(MethodCall|StaticCall $call): bool
         }
 
         $methodName = $this->nodeNameResolver->getName($call->name);
-        if (! str_starts_with((string) $methodName, 'assert') && ! in_array(
+        if (! str_starts_with((string) $methodName, 'assert') && ! ($call instanceof StaticCall && in_array(
```

Comment:
oh, it seems static call check is not needed, it just check the name, the StaticCall seems not needed then

Reply:
done :)